pkg/apis/provisioning/v1alpha5/constraints.go

```diff
@@ -51,30 +51,22 @@ func (c *Constraints) ValidatePod(pod *v1.Pod) error {
 	if err := c.Taints.Tolerates(pod); err != nil {
 		return err
 	}
-	// The constraints do not support this requirement
-	podRequirements := PodRequirements(pod)
-	for _, key := range podRequirements.Keys() {
-		if IgnoredLabels.Has(key) {
```

Comment:
Did you reimplement this for the new requirements

Reply:
This ignore label may change how we implement the validation. I need to get a better understanding the reason before I can implement it.

Reply:
Had an offline discussion with Brandon. Now the igore labels are excluded completely when we create the requriements.